src/platform_impl/web/web_sys/canvas.rs

```diff
@@ -239,6 +239,11 @@ impl Canvas {
                     event.prevent_default();
                 }
 
+                if event.key().len() > 1 {
+                    // ignore keypress for "Enter" key etc.
+                    return;
+                }
```

Comment:
Wouldn't this also throw out any "single character" key strings encoded as more than one byte? I suspect the only "correct" way of solving this might be to special-case all the "non-character" key strings specified in the [`uievents-key`](https://w3c.github.io/uievents-key/) spec.

Reply:
Maybe? The actual handler just does `event.chars().next().unwrap()`. So perhaps we should just check `event.key.chars().len()` here?

Reply:
Now it should support any multi-byte single character strings as well.

If there are multi-*character* strings however, those they will now be ignored, while previously, we would just take the first and ignore the rest.